### PR TITLE
Added the option to configure when the action should start reporting a new version

### LINKED ISSUE #880: Optional flag to check at Release Candidate phase
When a major WordPress release hits the initial Release Candidate phase (aka RC1), that's generally a good time to start testing for compatibility and then bumping the "tested up to" value.  It would be nice to have an optional flag or something with the action here to not check against the latest WordPress version but perhaps the latest WordPress version OR one that's in development and at least at the RC1 phase.

### PR DESCRIPTION
**Pull request by marekdedic**
Closes #880

### CODE REVIEW

**Comment by jeffpaul** [comment]
Nice!  It may be worth an update in `[README.md](https://github.com/skaut/wordpress-version-checker#readme)` as well to note the channel option.

**Comment by marekdedic** [comment]
Also, the roadmap:
- If this looks OK, I'll merge it and release it as a new minor version
- Once betas/RCs for WordPress 6.3 start rolling, I can add support for the beta channel (#915) and release that as a new minor version
- Concurrently with that, I'd like to switch the default channel to RC and release that as a new major version (#916)

File: README.md [hunk 1]
```diff
@@ -31,19 +31,48 @@ jobs:
...
-If you want the issues to be automatically assigned to someone, you can put their GitHub usernames in the config as well:
+```json
+{
+    "readme": ["path/to/first/readme.txt", "path/to/second/readme.txt"]
+}
+```
+
+### WordPress release channel
+
+By default, the app will only notify you once the new WordPress version is released (**This will change** - starting from version 2.0, the default value will be changed to `rc`). By setting the `channel` value to one of `stable` or `rc`, you can choose to be notified when the new version is fully release, is in the release candidate (RC) stage of development or when the first beta versions are released.
```
Comment (jeffpaul) [inline README.md:58]: Seems as though `beta` is also an option for `channel`?

Comment (marekdedic) [reply README.md:58]: I have temporarily disabled it in eaea4ffc4c17cf6c616aaf54c8786b7d0ad52ee9 - I have to wait until the betas start rolling again (late June I think) to implement the checks for them. RC should work as-is, though...

Comment (jeffpaul) [reply README.md:58]: Ok, seems good then, thanks!

Comment (marekdedic) [reply README.md:58]: Great, thanks for the second pair of eyes :)

**Pull request merged**